src/app/pages/MediaArticlePage/PagePromoSections/LastestMediaSection/index.tsx

```diff
@@ -0,0 +1,116 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { useContext } from 'react';
+import { jsx } from '@emotion/react';
+import path from 'ramda/src/path';
+import isEmpty from 'ramda/src/isEmpty';
+import useViewTracker from '#hooks/useViewTracker';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import generatePromoId from '../generatePromoId';
+import LatestMediaItem from './LatestMediaItem';
+import styles, {
+  StyledPromoItem,
+  StyledPromoList,
+  StyledSectionLabel,
+} from './index.styles';
+import {
+  LatestMedia,
+  TrackingBlock,
+  EventTrackingData,
+} from './LatestMediaTypes';
+
+const renderLatestMediaList = (
+  item: LatestMedia,
+  index: number,
+  eventTrackingData: EventTrackingData,
+  viewRef: () => Promise<void>,
+) => {
+  const ariaLabelledBy = generatePromoId({
+    sectionType: 'latest-media',
+    assetUri: null,
+    canonicalUrl: item.link,
+    uri: null,
+    contentType: item.type,
+    index,
+  });
+
+  return (
+    <StyledPromoItem key={index}>
```

Comment:
Similar to above, is it possible to use the `Promo` component directly and pass the css via the `css` prop?